yazi-plugin/preset/plugins/extract.lua

```diff
@@ -47,23 +57,45 @@ function M:try_with(url, pwd)
 		fail("Failed to determine a temporary directory for %s", url)
 	end
 
+	os.execute("mkdir -p " .. tostring(tmp))
```

Comment:
Please add a new method `fs.create(type, url)` above and similar to the `fs.remove()` method here and use it instead to achieve cross-platform compatibility https://github.com/sxyazi/yazi/blob/640e497c358c5ef576643e646cdb73be67bf7c52/yazi-plugin/src/fs/fs.rs#L36-L54

- type: available values `"file"`, `"dir"`
- url: Url to operate